src/TestSuite/Fixture/PHPUnitSubscriber/PHPUnitStartedSubscriber.php

```diff
@@ -0,0 +1,48 @@
+<?php
```

Comment:
Can we just put these into an extension namespace along with the above extension file?